components/camel-cmis/src/main/java/org/apache/camel/component/cmis/CMISSessionFacade.java

```diff
@@ -182,8 +182,9 @@ public Document getDocument(QueryResult queryResult) {
 
     public InputStream getContentStreamFor(QueryResult item) {
         Document document = getDocument(item);
-        if (document != null && document.getContentStream() != null) {
-            return document.getContentStream().getStream();
+        ContentStream contentStream = document.getContentStream();
```

Comment:
Won't it generate a NullPointerException when `document `is null ? Perhaps a 2 steps check or an Optional would make it.